Tests/DisplayLocationsTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+require_once "src/DisplayLocations.php";
+use PHPUnit\Framework\TestCase;
+
+
+
+class DisplayLocationsTest extends TestCase{
+    public function testDisplaylocations():void
```

Comment:
```suggestion
    public function testDisplaylocations_Success():void
```

What kind of test is this? naming it with what it is meant to show would be standard